.github/workflows/lint.yml

```diff
@@ -15,7 +16,7 @@ permissions:
   # pull-requests: read
 jobs:
   go:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04-16c-64g-600gb
```

Comment:
Not a huge deal, but I originally left these other jobs on the regular worker type since they shouldn't need as many resources, and I wasn't sure if they'd take up the runner for the duration of the linting.

Reply:
Good point!